### ISSUE TITLE
Creating and deleting shared private folders in Keybase filesystem under Windows

### ISSUE THREAD

**Issue by nuttyx** [keybase/client#3729]
First of all, I already had a "K:" drive in use, but the installation did not show any warning or error. My former drive was usable (it was also a dokan-powered cripto unit) but I coultn't find the private or public folder.
After unmounting the former drive and changing the letter, both folders under keybase were accesible without restarting at all. This is the second time a software I install "forces" me a fixed drive letter (both cases I already had that drive in use) and I find it somewhat annoying, since moving the former drive usually requires me more effort (software installation broken, image & music databases broken, etc.). Would it be possible to ask for a drive letter upon installation?

Moving into the bug itself: As I explained I'm using the Windows client, so commands to create and remove folders vary a little (md/rd instead of mkdir/rmdir on top of the different folder structure):
  md k:\private\someuser
  rd k:\private\someuser

That isn't a problem in itself (except most documentation is for *NIX), but IT IS a problem when trying to create or delete a "shared" folder:
  md k:\private\nuttyx,otheruser
will not create a "shared" folder for me and otheruser, but instead will "create" k:\private\nuttyx and another folder in my current path called "otheruser". If that path is k:\private it will raise an error in Keybase since you are not allowed to access the private folder of another user.
That is really a problem with Windows' syntax (correct syntax is with quotes: md "K:\private\nuttyx,otheruser"), but after creating a "shared" folder I tried to remove it and forgot again the quotes and deleted my own folder instead. After that, I recreated my folder via explorer.exe, but in the Keybase GUI my own folder (which I could see under K:\private) was not longer present.
I deleted it again and tried to re-create it via Keybase GUI -> New private folder, but nothing happened.
Then the GUI got messed up:

![screencap_1](https://cloud.githubusercontent.com/assets/20744741/17370780/ad388b78-599d-11e6-818a-8b73e4587d81.png)

The left menu stopped working and changing between public and private folder only changed background color from blue to white.
Also, my private view would no longer show my private folder, but the shared one even when both folders were present at k:\private in explorer.exe.
Restarting the system would fix the broken GUI, but my private folder wouldn't show up again in the GUI until I deleted and created it via rd and md in the keybase console. Also, deleting via explorer.exe the shared folder would not affect the GUI (it showed it as it was still present)

To summarize:

rd and md commands seem to work just fine to fix what you see in the Keybase GUI (until you break the GUI itself), but you might need to delete and recreate the folder.

It would be nice to make the comment about the need for quotes for md/rd commands for shared folders under Windows.

Keybase GUI seems to break when I try to create a new private folder or open the public folder.

Keybase GUI Version: 1.0.16-20160713110836+a1c00b3

**Comment by strib** [comment]
cc: @zanderz @MarcoPolo

**Comment by chrisnojima** [comment]
I think this is an artifact of a very old build. You should have a new build which resolves this issue.

**Comment by nuttyx** [comment]
My app is the latest released on keybase.io and no newer version has been released since I installed it.

**Comment by chrisnojima** [comment]
k, we're investigating. we have a build but it might be in the test channel

**Comment by johanbove** [comment]
@nuttyx remark on including the quotes in the Windows Shell when creating and removing shared folders is a very valid remark which I believe should be included in the Windows documentation.

**Comment by jsloan117** [comment]
Has this been resolved? I am not able to remove the shared folder I accidentally shared via the GUI not the CLI using the build in tools of Keybase itself. I'm able to remove the folder fine via Windows Explorer but it still shows up in Keybase app and gets recreated.

**Comment by zanderz** [comment]
This could also use further documentation, but drive letters don't work in the fs commands - you have to use /keybase/private/etc.

You can query and change drive letters with the `keybase kbfsmount` commands.

**Comment by jsloan117** [comment]
@zanderz thanks for the reply and the info. I guess I assume that related path was for non-windows only. While it appears I am able to ls the path fine, I am still unable to actually remove it. Is there any way I can help or something more you guys may need from me to further check it out?

`λ keybase fs ls /keybase/private/jsloan117,user2`
`λ keybase fs rm /keybase/private/jsloan117,user2`
`- ERROR  doesn't exist`

**Comment by zanderz** [comment]
Try `fs ls /keybase/private/"jsloan117,user2"` (quotes around the comma-separated usernames). Windows CMD did not seem to be a priority when this path naming system was designed.

**Comment by jsloan117** [comment]
Thanks again for the reply. I did try that after the two above I should have put that in there but it did return the exact same error.
